Package@swift-5.9.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:5.9
+import PackageDescription
+
+let package = Package(
+    name: "Ashton",
+    platforms: [.iOS("13.4"), .macOS(.v10_15), .visionOS(.v1)],
```

Comment:
This is a new file used only by Swift 5.9. Lower versions will keep using `Package.swift` and won’t see the visionOS platform.